### DISCUSSION TITLE
Js pokedex stats

### DISCUSSION THREAD

**Discussion by ArturoVelasquezG** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#408] [Q&A] [answered]
Hola
Disculpen, para mostrar las estadísticas del pokemon (hp, atk, def, etcétera.) ¿Qué función o comando se emplea para poder mostrarlo en html? llego a consultar desde el API pero no se como mostrarlo que no sea desde la consola.

**Comment by AngelCruzO** [comment, marked as answer]
con innerHTML. Creas tu variable para recoger el id del html, por ejemplo: pokeStats.innerHTML = data.stats...

**Reply by ArturoVelasquezG** [reply]
Muchas gracias. 😁

**Reply by AngelCruzO** [reply]
de nada

**Reply by MauricioGongoraLoria** [reply]
Raios tendras mas info de esto no logro sacar los datos de la consola...

**Reply by AngelCruzO** [reply]
@MauricioGongoraLoria https://developer.mozilla.org/es/docs/Web/API/Element/innerHTML

**Reply by MauricioGongoraLoria** [reply]
Muchas gracias lo revisare a ver que resulta!
